internal/controller/api_to_config.go

```diff
@@ -146,9 +146,21 @@ func routerToFRRConfig(r v1beta1.Router, alwaysBlock []frr.IncomingFilter, secre
 }
 
 func neighborToFRR(n v1beta1.Neighbor, prefixesInRouter []string, alwaysBlock []frr.IncomingFilter, routerVRF string, passwordSecrets map[string]corev1.Secret, bfdProfiles map[string]*frr.BFDProfile) (*frr.NeighborConfig, error) {
-	neighborFamily, err := ipfamily.ForAddresses(n.Address)
-	if err != nil {
-		return nil, fmt.Errorf("failed to find ipfamily for %s, %w", n.Address, err)
+	if n.Address == "" && n.Interface == "" {
+		return nil, fmt.Errorf("there is a neighbor with no interface and address")
```

Comment:
nit: let's say which neighbor

Reply:
how? both interface and address is empty, which field should I print ASN?

Reply:
yep, that for example

Reply:
I can do that . Fyi  @oribon  if I remember that error message was your feedback

Reply:
done